### PR TITLE
openshift-apps: add fedora-coreos-koji-tagger application

### PR DESCRIPTION
**Pull request by dustymabe**
original RFR: https://pagure.io/fedora-infrastructure/issue/7870

### CODE REVIEW

File: playbooks/openshift-apps/fedora-coreos-koji-tagger.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - mizdebsk
+    - jlebon
+
+# I need either this keytab role or to define the keytab in the
+# secret below not sure which
+  - role: openshift/keytab
+    app: fedora-coreos-koji-tagger
+    key: koji-keytab
+    secret_name: fedora-coreos-koji-tagger-coreos-bot-keytab
+    service: bodhi
```
Comment (mizdebsk) [inline playbooks/openshift-apps/fedora-coreos-koji-tagger.yml]: You can use `service: coreos-koji-tagger`

File: playbooks/openshift-apps/fedora-coreos-koji-tagger.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    host: 'idontknow'
```
Comment (mizdebsk) [inline playbooks/openshift-apps/fedora-coreos-koji-tagger.yml]: Use `host: "coreos-koji-tagger{{ env_suffix }}.fedoraproject.org"`

File: playbooks/openshift-apps/fedora-coreos-koji-tagger.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  - role: openshift/object
```
Comment (mizdebsk) [inline playbooks/openshift-apps/fedora-coreos-koji-tagger.yml]: You can embed secret into deployment config, separate secret.yaml shouldn't be needed AFAIK

**Review by mizdebsk — CHANGES_REQUESTED** [review]
Maybe call the role and playbook `coreos-koji-tagger` (without `fedora-` prefix) instead?
Some parts related to keytab need changing, otherwise looks good.

File: playbooks/openshift-apps/fedora-coreos-koji-tagger.yml (hunk 3 above)
Comment (dustymabe) [reply playbooks/openshift-apps/fedora-coreos-koji-tagger.yml]: +1 - done

File: playbooks/openshift-apps/fedora-coreos-koji-tagger.yml (hunk 2 above)
Comment (dustymabe) [reply playbooks/openshift-apps/fedora-coreos-koji-tagger.yml]: +1 - done

File: playbooks/openshift-apps/fedora-coreos-koji-tagger.yml (hunk 1 above)
Comment (dustymabe) [reply playbooks/openshift-apps/fedora-coreos-koji-tagger.yml]: +1 - done

**Comment by dustymabe** [comment]
> Maybe call the role and playbook `coreos-koji-tagger` (without `fedora-` prefix) instead?

renamed and updated the files

File: playbooks/openshift-apps/coreos-koji-tagger.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    - dustymabe
+    - jlebon
+    - mizdebsk
+
+  - role: openshift/keytab
+    app: coreos-koji-tagger
+    key: koji-keytab
+    secret_name: coreos-koji-tagger-coreos-bot-keytab
+    service: coreos-koji-tagger
+    host: host: "coreos-koji-tagger{{ env_suffix }}.fedoraproject.org"
```
Comment (mizdebsk) [inline playbooks/openshift-apps/coreos-koji-tagger.yml]: There is a typo - double "host"

**Review by mizdebsk — APPROVED** [review]
One typo, otherwise looks good.

File: playbooks/openshift-apps/coreos-koji-tagger.yml (hunk 4 above)
Comment (dustymabe) [reply playbooks/openshift-apps/coreos-koji-tagger.yml]: good catch. fixed now

**Review by mizdebsk — APPROVED** [review]
LGTM

**Comment by dustymabe** [comment]
rebased on top of latest master

**Comment by dustymabe** [comment]
https://infrastructure.fedoraproject.org/cgit/ansible.git/commit/?id=0999327fdaa03178403331ed10f8047eeaa9bd1d

**Pull request merged**